FWCore/Utilities/interface/ESGetToken.h

```diff
@@ -38,12 +40,29 @@ namespace edm {
     friend class eventsetup::EventSetupRecord;
 
   public:
-    ESGetToken() = default;
+    constexpr ESGetToken() noexcept = default ;
+
+    constexpr ESGetToken(ESGetToken<ESProduct,ESRecord> const&) noexcept = default;
+    constexpr ESGetToken(ESGetToken<ESProduct,ESRecord>&&) noexcept = default;
+
+    constexpr ESGetToken<ESProduct,ESRecord>& operator=(ESGetToken<ESProduct,ESRecord>&&) noexcept = default;
+    constexpr ESGetToken<ESProduct,ESRecord>& operator=(ESGetToken<ESProduct,ESRecord> const&) noexcept = default;
+
+    constexpr unsigned int transitionID() const noexcept { return m_transitionID;}
+    constexpr bool isInitialized() const noexcept { return transitionID() != std::numeric_limits<unsigned int>::max();}
```

Comment:
`EDGetToken` has `isUninitialized()`
https://github.com/cms-sw/cmssw/blob/7965eaafa4d3c9cbebf976d27f9ddad0bfbad27a/FWCore/Utilities/interface/EDGetToken.h#L47

Reply:
I actually think that `isInitialized()` would be more convenient to use, so maybe we should move `EDGetToken` to that instead?